test/helpers/utils.go

```diff
@@ -421,7 +421,7 @@ func getK8sSupportedConstraints(ciliumVersion string) (semver.Range, error) {
 	case IsCiliumV1_9(cst):
 		return versioncheck.MustCompile(">=1.12.0 <1.20.0"), nil
 	case IsCiliumV1_10(cst):
-		return versioncheck.MustCompile(">=1.16.0 <1.22.0"), nil
```

Comment:
This should be modified, what it actually needs to be done is adding `IsCiliumV1_11` and add `return versioncheck.MustCompile(">=1.17.0 <1.23.0"), nil`. Looking at this entire logic better it is also incorrect, the `case`s should be reversed, i.e., `IsCiliumV1_11(cst)` should be executed first and `IsCiliumV1_8(cst):` should be check last.